dash_auth/plotly_auth.py

```diff
@@ -155,6 +159,7 @@ def logout(self):
         invalidation_resp = requests.post(
             '{}{}'.format(api_requests.config('plotly_domain'),
                           '/o/revoke_token/'),
+            verify='DASH_STREAMBED_DIRECT_IP' not in os.environ,
```

Comment:
It doesn't appear that you're using `DASH_STREAMBED_DIRECT_IP` to connect to streambed here. As I said in https://github.com/plotly/dash-auth/pull/56#pullrequestreview-148690829, this is the _only_ case in which it's safe to make unverified requests. I'm happy to meet and discuss this if it's unclear...